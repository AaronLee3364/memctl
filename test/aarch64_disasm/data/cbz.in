fffffff00744ab54
b5000129 b4000161
