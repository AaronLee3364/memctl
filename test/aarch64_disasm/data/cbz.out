fffffff00744ab54  CBNZ    X9, #0xfffffff00744ab78
fffffff00744ab58  CBZ     X1, #0xfffffff00744ab84
